### ISSUE TITLE
compilation fails with openzwave 1.6

### ISSUE THREAD

**Issue by nouknouk** [OpenZWave/node-openzwave-shared#301]
Hi,

I got two compilation error with openzwave-1.6.737:

```
../src/utils.cc:255:49: error: ‘OpenZWave::Value’ has not been declared
   AddStringProp (nodeobj, type,      OpenZWave::Value::GetTypeNameFromEnum(value.GetType()));

../src/utils.cc:256:49: error: ‘OpenZWave::Value’ has not been declared
   AddStringProp (nodeobj, genre,     OpenZWave::Value::GetGenreNameFromEnum(value.GetGenre()));
```

I replaced the namespace for those two lines:

```
diff --git a/src/utils.cc b/src/utils.cc
index b5d8fa7..d46886c 100644
--- a/src/utils.cc
+++ b/src/utils.cc
@@ -252,8 +252,8 @@ namespace OZW {
                AddStringProp(nodeobj,  value_id,  buffer.c_str());
                AddIntegerProp(nodeobj, node_id,   value.GetNodeId());
                AddIntegerProp(nodeobj, class_id,  value.GetCommandClassId());
-               AddStringProp (nodeobj, type,      OpenZWave::Value::GetTypeNameFromEnum(value.GetType()));
-               AddStringProp (nodeobj, genre,     OpenZWave::Value::GetGenreNameFromEnum(value.GetGenre()));
+               AddStringProp (nodeobj, type,      OpenZWave::Internal::VC::Value::GetTypeNameFromEnum(value.GetType()));
+               AddStringProp (nodeobj, genre,     OpenZWave::Internal::VC::Value::GetGenreNameFromEnum(value.GetGenre()));
                AddIntegerProp(nodeobj, instance,  value.GetInstance());
                AddIntegerProp(nodeobj, index,     value.GetIndex());
                AddStringProp (nodeobj, label,     mgr->GetValueLabel(value).c_str());
```

... and it compiles now, but I don't know if it's an appropriate fix.

notes:
- openzwave-1.6.737
- node LTS v10.16.0
- gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04)
- node-openzwave-shared commit: `ee80492`

btw: thanks for your huge work on this lib

**Comment by nouknouk** [comment]
It seems related to a merge of yesterday in master branch of openzwave.
Cf https://github.com/OpenZWave/open-zwave/pull/1829/commits/e9273febf8ae7a627da13d77e07996d1b5050581

**Comment by Fishwaldo** [comment]
As implied by the namespace changes, Value Classes etc are internal implementation details and not intended to be used by applications directly.

I can see the value of those two methods though, and I will create a Public Method to get to them:
https://github.com/OpenZWave/open-zwave/issues/1831

**Comment by ekarak** [comment]
resolving - fixed with #302
